From 19584bed9588860047e3bd04e3fd2cf7e36adc3a Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Sat, 8 Jan 2022 11:28:37 +0000
Subject: [PATCH] Remove old -std=c++11 setting for Vc

This is set by ECM nowadays, and causes issues when a newer standard is needed (eg. for latest poppler)
---
 CMakeLists.txt | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3a86ec88f7d..9e87b9bda89 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -520,14 +520,6 @@ if (BUILD_VC)
 	    set(ADDITIONAL_VC_FLAGS "-Wabi -fabi-version=0 -ffp-contract=fast -fPIC")
 	endif()
 
-	#Handle Vc master
-	if(Vc_COMPILER_IS_GCC OR Vc_COMPILER_IS_CLANG)
-	    AddCompilerFlag("-std=c++11" _ok)
-	    if(NOT _ok)
-		AddCompilerFlag("-std=c++0x" _ok)
-	    endif()
-	endif()
-
 	macro(ko_compile_for_all_implementations_no_scalar _objs _src)
 	    if(PACKAGERS_BUILD)
 		vc_compile_for_all_implementations(${_objs} ${_src} FLAGS ${ADDITIONAL_VC_FLAGS} ONLY SSE2 SSSE3 SSE4_1 AVX AVX2+FMA+BMI2)
-- 
2.35.1

